tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -571,3 +574,148 @@ def test_invalid_interval(self):
                 },
             )
         assert response.status_code == 400
+
+    @mock.patch("django.utils.timezone.now")
+    def test_key_transaction_stats(self, mock_now):
+        mock_now.return_value = before_now().replace(tzinfo=pytz.utc)
+        prototype = {
+            "type": "transaction",
+            "transaction": "api.issue.delete",
+            "spans": [],
+            "contexts": {"trace": {"op": "foobar", "trace_id": "a" * 32, "span_id": "a" * 16}},
+            "tags": {"important": "yes"},
+        }
+        fixtures = (
+            ("d" * 32, before_now(minutes=32)),
+            ("e" * 32, before_now(hours=1, minutes=2)),
+            ("f" * 32, before_now(hours=1, minutes=35)),
+        )
+        for fixture in fixtures:
+            data = prototype.copy()
+            data["event_id"] = fixture[0]
+            data["timestamp"] = iso_format(fixture[1])
+            data["start_timestamp"] = iso_format(fixture[1] - timedelta(seconds=1))
+            self.store_event(data=data, project_id=self.project.id)
+
+        KeyTransaction.objects.create(
+            owner=self.authed_user,
+            organization=self.project.organization,
+            transaction=prototype["transaction"],
+            project=self.project,
+        )
+
+        with self.feature("organizations:discover-basic"):
+            response = self.client.get(
+                self.url,
+                format="json",
+                data={
+                    "end": iso_format(before_now()),
+                    "start": iso_format(before_now(hours=2)),
+                    "interval": "30m",
+                    "yAxis": "count()",
+                    "keyTransactions": True,
```

Comment:
Should we have a `stats` endpoint inside the other key-transactions endpoints? We might be overloading this endpoint quite a bit with key transactions, and the topN results that are coming.

Reply:
Created a separate endpoint for this, but they're still very similar functionally so I moved a bunch into the api base.